CalDavSynchronizer/CalDavSynchronizer.csproj

```diff
@@ -934,7 +958,6 @@
     <Resource Include="Resources\Import.png" />
     <Resource Include="Resources\ClearCache.png" />
     <None Include="Resources\ApplicationLogoLarge.png" />
-    <Content Include="Resources\LogoBanner.bmp" />
```

Comment:
Here `LogoBanner.bmp` is removed but `LogoBanner.png` is not added. Is this correct?

Reply:
yes, we don't use it in main project at all, just in installer